helpdesk-maker-checker-service/src/main/java/org/symphonyoss/symphony/bots/helpdesk/makerchecker/model/check/AgentExternalCheck.java

```diff
@@ -178,15 +179,30 @@ public boolean isCheckerType(MakerCheckerMessage makerCheckerMessage) {
     return symApprovedMessages;
   }
 
+  @Override
+  public void removeAttachmentAndDirectory(SymAttachmentInfo symAttachmentInfo) {
+    File diretorio = new File("/tmp/" + symAttachmentInfo.getId());
```

Comment:
Please rename this variable to 'directory'. Avoid using '/tmp' as base path, because it's not interoperable for different OS.

You should read the "java.io.tmpdir" system property to determine this path.

Reply:
Done.